docs/workflows.rst

```diff
@@ -28,7 +28,8 @@ All the commands should be run on Pulp 3 machine.
 
 
 2. Use the ``pulp_href`` of the created Migration Plan and its ``run/`` endpoint to run the
-migration.
+migration. Optionally, skip corrupted or missing Pulp 2 content by specifying the
+``skip_corrupted=True`` parameter.
```

Comment:
what about mentioning that  "with the next sync the missing content should be brought back" or remind user to run repo sync if skip corrupted content option has been used"

Reply:
I didn't want to overload the worflow section.
Maybe we need a page which will document all options in greater detail than it's done here.

Reply:
Added a more elaborate description to the REST API docs.